ci/README.md

```diff
@@ -59,3 +59,18 @@ fly -t <target-concourse> execute \
 
 - `BUCKET_NAME` should not contain `gs://`
 - `BACKUP_NAME` is the timestamped folder name containing the backup files and folders e.g. `2021-01-05T09:37:15_88808`
+
+## Purge expired sessions from Datastore
+
+Expired sessions can be purged from Datastore using the `purge_expired_sessions_from_datastore.yaml` task. This can be done via Concourse using the following command:
+
+```sh
+PROJECT_ID=<project_id> \
+DATAFLOW_TEMPLATE_VERSION=<dataflow_template_version> \
+EXPIRATION_TIME_OFFSET=<expiration_time_offset> \
+fly -t <target-concourse> execute \
+  --config ci/purge_expired_sessions_from_datastore.yaml
+```
+
+- `DATAFLOW_TEMPLATE_VERSION` is the template version on GCP you want to use
+- `EXPIRATION_TIME_OFFSET` is the offset in seconds expired sessions will be purged from (i.e 3600 would delete all sessions more than an hour old)
```

Comment:
Any reasons seconds were used or maybe not hours?. Either way, I think the variable should indicate if it is hours or seconds similar to how we do it with runner variables. It makes it less vague as when I look at the eq-pipelies repo variables file, I can't tell this is in seconds. Maybe `EXPIRATION_TIME_OFFSET_IN_SECONDS` for clarity.

Reply:
Thought about hours, but came to the conclusion that there might be a case where we might want to remove sessions closer to the actually time.

Reply:
Ya fair enough 👍